frontends/search/src/components/GroupPage.tsx

```diff
@@ -173,6 +174,8 @@ export const GroupPage = (props: GroupPageProps) => {
               setTotalPages(groupBookmarks.total_pages);
               setChunkMetadatas(groupBookmarks.chunks);
             });
+          } else if (response.status == 404) {
+            navigate("/group");
```

Comment:
I don't think we should forcibly navigate because that's confusing to the easier. Imo, we should just clear out the data.